lib/http.c

```diff
@@ -3283,10 +3283,13 @@ CURLcode Curl_http_size(struct Curl_easy *data)
   }
   else if(k->size != -1) {
     if(data->set.max_filesize &&
-       k->size > data->set.max_filesize) {
-      failf(data, "Maximum file size exceeded");
+       !k->ignorebody &&
+       (k->size > data->set.max_filesize)) {
+      failf(data, "Maximum **file size exceeded");
```

Comment:
```suggestion
      failf(data, "Maximum **file size exceeded");
```

the old text is better it doesn't need ***

Reply:
that was a mistake debug leftover